jobserver/management/commands/osi_release.py

```diff
@@ -83,10 +90,8 @@ def handle(self, *args, **options):
 
         tpp = Backend.objects.get(slug="tpp")
 
+        local_file = Path(options["report"])
```

Comment:
You can tell the parser to make this option a path using the `type` kwarg!  It's an `argparse` parser under the hood if that helps :)

Reply:
Ooh, yes that does. Thank you!